File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i, j=0;
        
        for(i =1;i<nums.size();i++)
        {
            if(nums[i-1] == nums[i])
                continue;
            else{
                nums[++j]=nums[i];
            }
        }
        return j+1;
    }
};

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int insertPointer = 1;
        for(int i = 1; i < nums.length; i++) {
            if(nums[i - 1] != nums[i]) {
                nums[insertPointer] = nums[i];
                insertPointer++;
            }
        }
        
        return insertPointer;
    }
}